arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -1203,403 +809,284 @@ private void handleCreate(Address target) {
-
-          if (hasCode) {
-            final SmartContractCallSection section =
-                new SmartContractCallSection(
-                    this, myAccountSnapshot, calledAccountSnapshot, rawCalledAddress, imcFragment);
-            this.addTraceSection(section);
-            this.currentFrame().sectionToUnlatch(section);
-          } else {
-            //
-            // CALL EXECUTED
-            //
-
-            // TODO: fill the callee & requested return data for the current call frame
-            // TODO: i.e. ensure that the precompile frame behaves as expected
-
-            Optional<PrecompileInvocation> precompileInvocation =
-                targetPrecompile.map(p -> PrecompileInvocation.of(this, p));
-
-            // TODO: this is ugly, and surely not at the right place. It should provide the
-            // precompile result (from the precompile module)
-            // TODO useless (and potentially dangerous) if the precompile is a failure
-            if (targetPrecompile.isPresent()) {
-              this.callStack.newPrecompileResult(
-                  this.stamp(), Bytes.EMPTY, 0, targetPrecompile.get().address);
-            }
-
-            final NoCodeCallSection section =
-                new NoCodeCallSection(
-                    this,
-                    precompileInvocation,
-                    myAccountSnapshot,
-                    calledAccountSnapshot,
-                    rawCalledAddress,
-                    imcFragment);
-            this.addTraceSection(section);
-            this.currentFrame().sectionToUnlatch(section);
-          }
-        }
-      }
+      case CREATE -> new CreateSection(this);
 
-      case JUMP -> {
-        AccountSnapshot codeAccountSnapshot =
-            AccountSnapshot.fromAccount(
-                frame.getWorldUpdater().get(this.currentFrame().codeAddress()),
-                true,
-                this.transients
-                    .conflation()
-                    .deploymentInfo()
-                    .number(this.currentFrame().codeAddress()),
-                this.currentFrame().underDeployment());
-
-        JumpSection jumpSection =
-            new JumpSection(
-                this,
-                ContextFragment.readContextData(callStack),
-                this.factories.accountFragment().make(codeAccountSnapshot, codeAccountSnapshot),
-                ImcFragment.forOpcode(this, frame));
+      case CALL -> new CallSection(this);
 
-        this.addTraceSection(jumpSection);
-      }
+      case JUMP -> new JumpSection(this);
     }
+  }
 
-    // In all cases, add a context fragment if an exception occurred
-    if (Exceptions.any(this.pch().exceptions())) {
-      this.currentTraceSection()
-          .addFragment(
-              this, this.currentFrame(), ContextFragment.executionEmptyReturnData(callStack));
-    }
+  /**
+   * 0x 0000000000000000000000000000000000000000000000000000000000000020
```

Comment:
@OlivierBBB What is it ? Was it for debuging purpose and can now be deleted ?

Reply:
It seems like very old stuff (debugging return data, so maybe as old as Tbilissi ?)